### PR TITLE
HL-95, 116: Application attachments to api

### PR DESCRIPTION
**Pull request by melniiv**
## Description :sparkles:
- attachments view, add, delete + error handling

## Additional notes :spiral_notepad:
-todo: implement the file name when attachment name field is added to the model

### CODE REVIEW

File: frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx [hunk 1]
```diff
@@ -43,9 +69,23 @@ const AttachmentsList: React.FC<AttachmentsListProps> = ({
...
+      <$UploadContainer onClick={handleUploadClick}>
+        <$PrimaryButton
+          isLoading={isUploading}
+          loadingText={t(`common:upload.isUploading`)}
+          style={{ width: 'auto' }}
+          iconLeft={<IconPlus />}
+        >
+          {t(`${translationsBase}.add`)}
+        </$PrimaryButton>
+        <input
```
Comment (Nizar-Rahme) [inline frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx:81]: So you decided to not use the HDS component, or we will replace this later?

**Review by Nizar-Rahme — APPROVED** [review]
LGTM! Well done 👌

File: frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx (hunk 1 above)
Comment (melniiv) [reply frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx:81]: its not implemented yet:)

Comment (melniiv) [reply frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx:81]: and i checked it, its not what we need actually.... but lets see if we can reuse it at some point

**Pull request merged**